app/controllers/enterprises_controller.rb

```diff
@@ -39,6 +39,10 @@ def check_permalink
 
   private
 
+  def set_enterprise
+    @enterprise = Enterprise.find(params[:id])
```

Comment:
what if the enterprise does not exist? do we need any authorization ability here?

Reply:
@enricostano I have changed this to `Enterprise.find_by_id` so that an error isn't raised when a bad `id` is provided. This method is only used by `EnterprisesController#relatives`, logic there already deals with the case that no enterprise is found.

No authorisation required here IMO, as this is just describing the existence of relationships in the network. I think this is fine to be public knowledge.